ports/linmath/portfile.cmake

```
@@ -0,0 +1,13 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO datenwolf/linmath.h
+    REF refs/heads/master/0538757

```

```suggestion
    REF 3954f97604a522c4b9c0268188f1a3d47ad2a205
```
This would be the correct way

---

Oh okay. Where did you get that?

---

Okay I checked It is the permalink.

---

For reference: https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_from_github.md

---

Branches are expected to move over time. So the SHA checksum would no longer match. Tags are expected to be stable names.

---

I think it's good.

---

This should be replaced with the stable reference.